core/templates/css/oppia.css

```diff
@@ -3264,35 +3264,41 @@ md-card.preview-conversation-skin-supplemental-card {
 /* New Splash page */
 
 .oppia-splash-section-one {
```

Comment:
[Optional] From here (line 3266) to line 3465 seems to be specific for the splash page. Can we move these styles to the Splash page component?

Reply:
Done